base/threadcall.jl

```diff
@@ -44,7 +44,7 @@ macro threadcall(f, rettype, argtypes, argvals...)
     body = wrapper.args[2].args
     args = Symbol[]
     for (i,T) in enumerate(argtypes)
-        arg = symbol("arg$i")
+        arg = Symbol("arg$i")
```

Comment:
Unfortunately `Symbol(1::Int)` doesn't work here, same with `Symbol('a'::Char)` (suggested in the `[2:end]` -> `[2]` comment above).

Perhaps these should be defined? They are for `string`.